gcloud-java-logging/src/main/java/com/google/gcloud/logging/testing/LocalLoggingHelper.java

```diff
@@ -0,0 +1,77 @@
+package com.google.gcloud.logging.testing;
+
+import com.google.gcloud.pubsub.spi.v2.testing.LocalLoggingImpl;
+import com.google.logging.v2.LoggingServiceV2Grpc;
+
+import io.grpc.ManagedChannel;
+import io.grpc.Server;
+import io.grpc.netty.NegotiationType;
+import io.grpc.netty.NettyChannelBuilder;
+import io.grpc.netty.NettyServerBuilder;
+import io.netty.channel.local.LocalAddress;
+import io.netty.channel.local.LocalChannel;
+import io.netty.channel.local.LocalServerChannel;
+
+import java.io.IOException;
+import java.net.SocketAddress;
+
+public class LocalLoggingHelper {
```

Comment:
I don't think `LocalLoggingHelper` (and its associated comments) is part of #472.
#472 is for providing an alternative for a local fake implementation (`LocalLoggingImpl`).
Like in other services we provide the user (and also use it in our testing) a helper (like this one)
to *get* a local services (which could come from SDK, public location or local implementation).